lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -145,4 +145,26 @@ def table_alias_length
     63
   end
   alias index_name_length table_alias_length
+
+
+  private
+
+  # Extracts the value from a PostgreSQL column default definition.
+  # Some of postgresql's default regexps are not going to work with cockroachdb,
+  # leaving the default values out of the schema.rb file.
+  def extract_value_from_default(oid, default) # :nodoc:
+    ret = super
+    if ret.nil?
+      ret = case default
+        # Numeric types are identified in a different way by cockroachdb
+        when /\A\(?(-?\d+(\.\d*)?)\)?(:::INT)?\z/
```

Comment:
A few remarks, you might be able to answer them as I have not looked into the details of how this works.
- Is this input string lowercased earlier on? If not, I might suggest adding the lowercase regex modifier here. I assume this because postgres uses `bigint` lower case.
- We also have sized types such as `INT{4,8,16,32,64}` and `DECIMAL`. Do we also want to extract the values from those? Considering this is just considered with extracting values, disregarding the types, I think this is appropriate.